modules/ding_library/plugins/content_types/library_list.inc

```diff
@@ -47,6 +47,7 @@ function ding_library_library_list_content_type_render($subtype, $conf, $panel_a
 
   // Get all active library title and nid's.
   $nodes = db_select('node', 'n')
+    ->addTag('library_list')
```

Comment:
Should be renamed to `'language') or similar instead.

Reply:
Don't see the need in this. If in the future we will have to make other alterations on this query which are not related to language, so then it will be confusing.

Reply:
This `$query->condition('language', $language->language);` is used multiple times. Here you define a function for it. Why not re-use that? The "library_list" naming is not saying for what this function actually does.